src/Application.vue

```diff
@@ -375,6 +375,24 @@ export default {
         this.dataToSave.widgetInstances = themeSavedWidgetInstances
       }
 
+      if(_.has(this.dataToSave.values, 'containerBackgroundType')) {
```

Comment:
@OleksandrLavrynenko I think this `if` is unnecessary, especially when you have a `default` case in the `switch`. Can you confirm?